pipeline/ingestor/ingestor_works/src/main/scala/weco/pipeline/ingestor/works/models/IndexedWork.scala

```diff
@@ -0,0 +1,32 @@
+package weco.pipeline.ingestor.works.models
+
+import weco.catalogue.internal_model.identifiers.{DataState, IdState}
+import weco.catalogue.internal_model.work.{WorkData, WorkState}
+
+sealed trait IndexedWork
+
+object IndexedWork {
+  case class Visible(
+    data: WorkData[DataState.Identified],
+    state: WorkState.Indexed,
+    debug: DebugInformation.Visible,
+    display: DisplayWork
+  ) extends IndexedWork
+
+  case class Redirected(
+    redirectTarget: IdState.Identified,
```

Comment:
All of these have state and debug, then some other parameters.  I think it would be nice to have those two first, then the other parameters after, for consistency.

Reply:
👍 I wrote it this way because originally I had newlines to mark where I'd forked form the Work[WorkState.Indexed] model, but apparently autoformatting has removed those. I'll rearrange them.